src/extension/printing/pom.xml

```diff
@@ -35,11 +35,6 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>xml-apis</groupId>
-      <artifactId>xml-apis-xerces</artifactId>
-      <version>2.7.1</version>
-    </dependency>
     <dependency>
       <groupId>xerces</groupId>
       <artifactId>xercesImpl</artifactId>
```

Comment:
xercesImpl could probably be removed as well

Reply:
I don't know enough about the module as to make an informed decision